package.json

```diff
@@ -55,6 +55,7 @@
         "onCommand:aws.stepfunctions.publishStateMachine",
         "onView:aws.cdk.explorer",
         "onCommand:aws.refreshCdkExplorer",
+        "onCommand:aws.cdk.renderStateMachineGraphFromCDK",
```

Comment:
The command is already prefixed with `aws.cdk`, so we can probably leave out the `FromCDK` bit.